test_remove_due_date.py

```diff
@@ -0,0 +1,25 @@
+from todo import TodoList
+from pytest import fixture
+
+@fixture
+def myList():
+    #Arrange
+    myList = TodoList()
+    myList.tasks = [("Buy grocery", "Medium", "2024-12-2", [], "Siyuan", False)]
+    return myList
+
+
+def test_remove_due_date(capfd, myList):
+    #Act
+    myList.remove_due_date(1)
+    
+    #Assert
```

Comment:
Missing check that the due date is actually removed. These asserts just verify console output.